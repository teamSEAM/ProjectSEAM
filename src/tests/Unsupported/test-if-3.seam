#TESTING OPERATIONS#
entity World:
	func start():
		screen.init(100,100)
		if(1==1):
			screen.out("== OK")
		else:
			screen.out("== FAIL")
		if(2>1):
			screen.out("2>1 OK")
		else:
			screen.out("2>1 FAIL")
		if(1>=1):
			screen.out("1>=1 OK")
		else:
			screen.out("1>=1 FAIL")
		if(2>=1):
			screen.out("2>=1 OK")
		else:
			screen.out("2>=1 FAIL")
		if(1<=1):
			screen.out("1<=1 OK")
		else:
			screen.out("1<=1 FAIL")
		if(1<=2):
			screen.out("1<=2 OK")
		else:
			screen.out("1<=2 FAIL")
		if(1<2):
			screen.out("1<2 OK")
		else:
			screen.out("1<2 FAIL")
