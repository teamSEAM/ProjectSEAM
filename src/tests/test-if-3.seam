#TESTING OPERATIONS#
entity World:
	func start():
		if(1==1):
			screen.out("== OK")
		else:
			screen.out("== FAIL")
		if(2>1):
			screen.out("2>1 OK")
		else:
			screen.out("2>1 FAIL")
		if(1>=1):
			screen.out("1>=1 OK")
		else:
			screen.out("1>=1 FAIL")
		if(2>=1):
			screen.out("2>=1 OK")
		else:
			screen.out("2>=1 FAIL")
		if(1<=1):
			screen.out("1<=1 OK")
		else:
			screen.out("1<=1 FAIL")
		if(1<=2):
			screen.out("1<=2 OK")
		else:
			screen.out("1<=2 FAIL")
		if(1<2):
			screen.out("1<2 OK")
		else:
			screen.out("1<2 FAIL")
