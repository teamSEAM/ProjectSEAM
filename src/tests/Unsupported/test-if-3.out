== OK
2>1 OK
1>=1 OK
2>=1 OK
1<=1 OK
1<=2 OK
1<2 OK

